pkg/inspection/metadata/form/form.go

```diff
@@ -20,36 +20,79 @@ import (
 	"sync"
 
 	"github.com/GoogleCloudPlatform/khi/pkg/inspection/metadata"
+	"github.com/GoogleCloudPlatform/khi/pkg/server/upload"
 	"github.com/GoogleCloudPlatform/khi/pkg/task"
 )
 
 const FormFieldSetMetadataKey = "form"
 
-type FormFieldHintType string
+// ParameterInputType represents the type of parameter form field.
+type ParameterInputType string
 
 const (
-	HintTypeWarning = "warning"
-	HintTypeInfo    = "info"
+	Group ParameterInputType = "group"
+	Text  ParameterInputType = "text"
+	File  ParameterInputType = "file"
 )
 
-type FormField struct {
-	Priority        int               `json:"-"`
-	Id              string            `json:"id"`
-	Type            string            `json:"type"`
-	Label           string            `json:"label"`
-	Description     string            `json:"description"`
-	Hint            string            `json:"hint"`
-	HintType        FormFieldHintType `json:"hintType"`
-	Default         string            `json:"default"`
-	AllowEdit       bool              `json:"allowEdit"`
-	Suggestions     []string          `json:"suggestions"`
-	ValidationError string            `json:"validationError"`
+// ParameterHintType represents the types of hint message shown at the bottom of parameter forms.
+type ParameterHintType string
+
+const (
```

Comment:
Could you add comments to this Enum? Without comments, it's unclear how each value is used in the frontend. `None` and `Info` are very similar.

- None:  The user can use the form as is.  Use this value when there are no particular hints.
- Info:  The user can use the form as is, but there are helpful hints for input (e.g., the calculated time range or query).
- Warn: The user can use the form as is, but there are points to be aware of when entering input (e.g., the entered cluster not found).
- Error: The user cannot use the form as is.  Prompt the user to make corrections based on the value of the hint.

Reply:
fixed